edge-modules/TestAnalyzer/PartitionReceiverHandler.cs

```diff
@@ -15,7 +15,7 @@ class PartitionReceiveHandler : IPartitionReceiveHandler
         const string SequenceNumberPropertyName = "sequenceNumber";
         const string EnqueuedTimePropertyName = "iothub-enqueuedtime";
         const string BatchIdPropertyName = "batchId";
-        static readonly ILogger Log = Logger.Factory.CreateLogger<PartitionReceiveHandler>();
+        static readonly ILogger Logger = ModuleUtil.CreateLogger(nameof(PartitionReceiveHandler));
```

Comment:
Revert this change.

Reply:
For context, which part of this do you want to revert? ModuleUtil, nameof, or both? Both these patterns are replicated elsewhere so I will need to make more changes.

Reply:
Assuming the nameof, judging by your other comment.

Reply:
I will revert it back to use Logger.Factory.CreateLogger<PartitionReceiveHandler>(); and keep variable name "Logger".

Reply:
ignore this comment; since you wrapped some logging logic inside this new ModuleUtil.CreateLogger method; and it doesn't support generic type argument.